### PR TITLE
add icon-font | changed all icon classes to new icon-font

### PR DESCRIPTION
**Pull request by witflash**
1. Created custom font `fontello` with icons which contained in project.
2. Changed all icon classes in all headers to new font-icon.
3. Some fixes in header-5 (repaired buttons `angle-left` & `angle-right`).

### CODE REVIEW

File: app/styles/headers/_fifth.scss [hunk 1]
```diff
@@ -113,9 +113,10 @@
         transform: translateY(-50%);
 
         i {
-            position: absolute;
```
Comment (Oleksandr58) [inline app/styles/headers/_fifth.scss:116]: 1. How without `position:absolute` you shift right arrow ?
2. This arrows in the end html - they don't put in the right place without `position:absolute` and be after all blocks.

File: app/styles/headers/_eighth.scss [hunk 2]
```diff
@@ -126,7 +126,7 @@ $button_bg: #f78d78;
 
             .icon{
                 position: absolute;
-                left: -35px;
+                left: -45px;
```
Comment (Oleksandr58) [inline app/styles/headers/_eighth.scss:129]: Don't understand why this's important, but ok.

**Review by Oleksandr58 — CHANGES_REQUESTED** [review]

File: app/styles/headers/_fifth.scss (hunk 1 above)
Comment (Oleksandr58) [reply app/styles/headers/_fifth.scss:116]: Test approved - all good.
My mistake.

**Review by Oleksandr58 — APPROVED** [review]

File: app/styles/headers/_eighth.scss (hunk 2 above)
Comment (witflash) [reply app/styles/headers/_eighth.scss:129]: This adds more space between icon and text. It's not imortant but only for more beauty :)

Comment (Oleksandr58) [reply app/styles/headers/_eighth.scss:129]: Then I agree.
Beaty is important!)

**Pull request merged**